api/v1alpha8/types.go

```diff
@@ -156,6 +162,7 @@ type BastionStatus struct {
 	IP                  string                     `json:"ip,omitempty"`
 	FloatingIP          string                     `json:"floatingIP,omitempty"`
 	ReferencedResources ReferencedMachineResources `json:"referencedResources,omitempty"`
+	PortsStatus         []PortStatus               `json:"portsStatus,omitempty"`
```

Comment:
@mdbooth I think you had an idea of a similar struct than `ReferencedResources` where we could regroup things like `PortsStatus`. What were the name ideas? Something like `AdoptedResources`?

Reply:
I was thinking `DependentResources`.

`ReferencedResources`: must exist and be resolved before creation. Not deleted with the machine.
`DependentResources`: created before the machine and deleted with the machine. Must be checked for adoption before either creation or deletion.

Reply:
cool, works for me.